spec/proxy.spec.js

```diff
@@ -242,7 +252,7 @@ describe('proxy', function() {
                     [methodName]('/https://example.com/blah')
                     .expect(200)
                     .expect(function() {
-                        expect(fakeRequest.calls.argsFor(0)[0].proxy).toBe('http://proxy/');
+                        expect(fakeRequest.calls.argsFor(0)[0].agent.http.proxy).toEqual(expectedProxy);
```

Comment:
Only changing `.proxy` into `.agent.http.proxy` made the tests fail with
```
Expected http://proxy/ to be 'http://proxy/'
```
And I have no idea why.